cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -35,13 +35,14 @@ func myHostPort() string {
 	// 	http://whatismyipaddress.com/api
 	// 	http://ipinfo.io/ip
 	ip := strings.TrimSpace(httpGet("https://api.ipify.org/?format=text"))
-	return ip + ":" + defaultRPCPort
+	return "https://" + ip + ":" + defaultRPCPort
```

Comment:
@thomshutt I don't know why GH shows this change. Could you please fix it before merging?